test/test_pytree.py

```diff
@@ -543,6 +543,7 @@ def __init__(self, x, y):
             DummyType,
             lambda dummy: ([dummy.x, dummy.y], None),
             lambda xs, _: DummyType(*xs),
+            type_fqn="test_pytree_custom_type_serialize.DummyType",
```

Comment:
Why doesn't python pickle have this problem?

Reply:
it does